community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -494,6 +494,9 @@
     @Description( "Log allocated bytes for executed queries that took longer than the configured threshold." )
     public static final Setting<Boolean> log_queries_allocation_logging_enabled = setting( "dbms.logs.query.allocation_logging_enabled", BOOLEAN, FALSE );
 
+    @Description( "Log page hits and page faults for executed queries that took longer than the configured threshold." )
+    public static final Setting<Boolean> log_queries_page_detail_logging_enabled = setting( "dbms.logs.query.page_logging_enabled", BOOLEAN, FALSE );
```

Comment:
I can't see these log settings in neo4j.conf. It would be useful to add them as commented out defaults.